src/components/AccountAddresses.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="wallet-identifier">
+        <div v-for="n in 6" class="account" :key="n" :class="{ placeholder: n > addresses.length, 'pop-in': animate }">
+            <Identicon v-if="n <= addresses.length" :address="addresses[n - 1]"></Identicon>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import Identicon from './Identicon.vue';
+
+@Component({components: { Identicon }})
+export default class WalletIdentifier extends Vue {
+    @Prop({ default: () => [], type: Array })
+    public addresses!: string[];
+
+    @Prop({ default: false, type: Boolean })
+    public animate!: boolean;
+}
+</script>
+
+<style scoped>
+    .wallet-identifier {
+        position: relative;
+        --hex-width: 4rem;
```

Comment:
The size should rather be relative to the width of the component, to be able to make the whole component bigger and smaller.

The component can have a default width/height, but it should adapt to externally set width/height.

Reply:
Suggestion: container size could have fixed aspect ratio depending on width with css padding-top fixed aspect ratio trick. Then, all widths could be percentual tothe container width and all heights percentual to the container height.
That's the simplet css only solution I can think of right now.
Are there other / better approaches?

Reply:
I cannot motivate myself to to this right now. I don't see the sense in doing lots of calculations and experiments and basically do this again from scratch just to trade one bad solution for another. Or am I too stupid to see how easy this is? :/

Reply:
I think it's ok for now. The size can be adapted by setting `--hex-width`.
For the automatic resizing I have a solution in mind.
However, I will not get to implement that anytime soon, so I opened an issue here: https://github.com/nimiq/vue-components/issues/21